src/data/monsters.txt

```diff
@@ -2175,13 +2175,17 @@ stumbling-drunk congoer (female)	1972	congoerf1.gif,congoerf2.gif,congoerf3.gif
 stumbling-drunk congoer (male)	1969	congoer1.gif,congoer2.gif,congoer3.gif	Scale: 5 Init: -10000 P: dude
 swimming pool monster	1970	poolmonster.gif	Scale: 5 Init: -10000 P: elemental
 
-# Crimbo 2018
+# Crimbo 2021
```

Comment:
I think you've accidentally switched 2018 and 2021

Reply:
Sigh.